Description: Do not use intersphinx
 Avoids network use when building sphinx docs.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-21

--- oslo.messaging-1.3.0~a9.orig/doc/source/conf.py
+++ oslo.messaging-1.3.0~a9/doc/source/conf.py
@@ -8,7 +8,7 @@ sys.path.insert(0, os.path.abspath('../.
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx', 'oslosphinx']
+extensions = ['sphinx.ext.autodoc', 'oslosphinx']
 
 # autodoc generation is a bit aggressive and a nuisance when doing heavy
 # text edit cycles.
